.github/workflows/pipeline.yml

```diff
@@ -164,9 +164,21 @@ jobs:
         id: check
         run: echo "is_enabled=${{ secrets.DOCKER_HUB_USERNAME != '' }}" >> $GITHUB_OUTPUT
 
+  lower-repo-name:
```

Comment:
I think it is better to have a single step duplicated in the `build` and `push-manifest` jobs, than having a whole new job just to do the lowercasing. IMO it would be a waste of resources.

Can't you just add the step in each job?

Reply:
In my initial idea, I did the same thing, but I felt it was not elegant enough to repeat the same thing in two jobs, so I added a new job to handle this task uniformly.